### ISSUE TITLE
Release 1.9

### ISSUE THREAD

**Issue by foo4u** [emicklei/rendersnake#4]
Hi @emicklei 

Do you have any plans to do a release soon with the latest changes since 1.8?

Thanks in advance,
Scott

**Comment by emicklei** [comment]
Hi Scott,

Yes, I will make a new release 1.9 asap.

On Fri, Feb 13, 2015 at 11:13 AM, Scott Rossillo <notifications@github.com>
wrote:

> Hi @emicklei <https://github.com/emicklei>
>
> Do you have any plans to do a release soon with the latest changes since
> 1.8?
>
> Thanks in advance,
> Scott
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emicklei/rendersnake/issues/4>.
>



-- 
^ Ernest Micklei

http://ernestmicklei.com

**Comment by emicklei** [comment]
The new release has been submitted to the staging repo of sonatype. It will take some time before it is available in Maven Central.

**Comment by foo4u** [comment]
Awesome, you rock!

**Comment by foo4u** [comment]
@emicklei sorry to bug you. 1.9 doesn't seem to have hit [Maven Central][0] yet. Mind taking a look?

Thanks!

[0]: http://search.maven.org/#search%7Cga%7C1%7Crendersnake

**Comment by emicklei** [comment]
i just did,  forgot to sign the artifacts first. will redo the deployment

**Comment by emicklei** [comment]
resumed working on it ; somehow the signatures files (.asc) are not uploaded. fighting the plugins....

**Comment by emicklei** [comment]
turns out that the Javadocs are in a bad shape; the latest maven plugin complains a lot and without docs, the sonatype repo will not accept the new release. I may take some time to fix and since I am leaving for a week holiday, I am sorry to report that it will be finished after that.

**Comment by emicklei** [comment]
http://stackoverflow.com/questions/15886209/maven-is-not-working-in-java-8-when-javadoc-tags-are-incomplete

**Comment by emicklei** [comment]
i am stuck.  have to leave now :-(

![screen shot 2015-02-20 at 11 13 34](https://cloud.githubusercontent.com/assets/686220/6284565/8eba02ba-b8f1-11e4-847e-b7e4843ad7f5.png)

**Comment by foo4u** [comment]
Thanks for the update. Are you building with Java 8? There's a lot of known issues with the Javadoc linter being much stricter than previous versions and causing problems. JetS3 [updated their build file][0] to get around this.

Either way, I can fix some linter issues why you're away and send a PR.

[0]: https://bitbucket.org/jmurty/jets3t/commits/6c8c50bcf1a2d4da4681154615f914e801182204

**Comment by emicklei** [comment]
yes, i found a workaround for the time being but somehow the verification
of the signed artifact fails.

Much appreciated if you can help with the lint issues.

On Friday, February 20, 2015, Scott Rossillo <notifications@github.com>
wrote:

> Thanks for the update. Are you building with Java 8? There's a lot of
> known issues with the Javadoc linter being much stricter than previous
> versions and causing problems. JetS3 updated their build file
> <https://bitbucket.org/jmurty/jets3t/commits/6c8c50bcf1a2d4da4681154615f914e801182204>
> to get around this.
>
> Either way, I can fix some linter issues why you're away and send a PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emicklei/rendersnake/issues/4#issuecomment-75262390>.
>


-- 
^ Ernest Micklei

http://ernestmicklei.com

**Comment by emicklei** [comment]
Seems that more people are having problems. 
https://issues.sonatype.org/browse/OSSRH-5675

I am now considering using Bintray instead.

**Comment by emicklei** [comment]
please try fetching the new release using the following repo
```
        <repository>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
            <id>emicklei</id>
            <name>emicklei</name>
            <url>http://dl.bintray.com/emicklei/maven</url>
        </repository>
```

**Comment by foo4u** [comment]
:+1: works for me!

**Comment by talios** [comment]
Can this be released to Maven Central please.  The problems with Javadoc were fixed in later releases of the javadoc plugin ( and in JDK 8 patch releases themselves ).

**Comment by emicklei** [comment]
I will give it another try this weekend.

**Comment by emicklei** [comment]
first retry no luck. some signature files are missing.

```
failureMessage	Missing Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9-test-sources.jar.asc' does not exist for 'rendersnake-1.9-test-sources.jar'.
failureMessage	Missing Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9.jar.asc' does not exist for 'rendersnake-1.9.jar'.
failureMessage	Missing Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9.pom.asc' does not exist for 'rendersnake-1.9.pom'.
failureMessage	Missing Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9-sources.jar.asc' does not exist for 'rendersnake-1.9-sources.jar'.
failureMessage	Missing Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9-javadoc.jar.asc' does not exist for 'rendersnake-1.9-javadoc.jar'.
```

i probably need to run the javadocs BEFORE deploy

**Comment by emicklei** [comment]
so, i got the javadoc (silent xdoclint) and signing part fixed. Now they are generated and uploaded as part of the staging release.

Unfortunately, only 2 out of 5 are found incorrect. Have to find out why...

```
[ERROR]     * Invalid Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9.jar.asc' is not a valid signature for 'rendersnake-1.9.jar'.
[ERROR]     * Invalid Signature: '/org/rendersnake/rendersnake/1.9/rendersnake-1.9-javadoc.jar.asc' is not a valid signature for 'rendersnake-1.9-javadoc.jar'.
```

**Comment by emicklei** [comment]
```
[INFO] Remote staging repositories released.
```

now it should appear in Maven Central any time soon....

**Comment by talios** [comment]
+1000 excellent - you rock.

**Comment by kaushikk** [comment]
i was able to add it as a dependency using Android Studio!!!! Great.... Thanks!!!

**Comment by talios** [comment]
Finally got around to testing this - um, I see rendersnake-1.9 in Maven Central, but not the parent...

    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal on project com.smxemail.rest.ramlprovider: Could not resolve dependencies
    for project com.smxemail:com.smxemail.rest.ramlprovider:maven-plugin:1.0-39b2c9a051da3172bebe9116557133da69b30f23: 
    Failed to collect dependencies at org.rendersnake:rendersnake:jar:1.9: Failed to read artifact descriptor for
     org.rendersnake:rendersnake:jar:1.9: Could not find artifact org.rendersnake:rendersnake-parent:pom:1.9 in
      Nexus (http://nexus.smxemail.com/nexus/content/groups/public/) -> [Help 1]

http://search.maven.org/#search%7Cga%7C1%7Crendersnake shows it not there...

**Comment by emicklei** [comment]
ic. will fix that.  

given the long story for getting the current artifact in central, I am strongly considering simplifying the rendersnake project itself such that a parent is not needed. Agree?

**Comment by emicklei** [comment]
I just released 1.9.0  that does not require the parent pom.
